### ISSUE TITLE
React should be a peer dependency instead of a dependency in package.json?

### ISSUE THREAD

**Issue by K3vinF** [Palmabit-IT/react-cookie-law#15]
I see that React is listed as a dependency in package.json.
It looks like this causes React code te be injected into the generated javascript in the /dist of the npm module.
I don't think this is needed (nor desirable)?

I think it would be better to list React as a peer dependency?

**Comment by alessandro231** [comment]
Yes, better.  I'll move the dependency to the peer dependency with the next release.
Thanks!
